test/js/ez-platform-ui-app.js

```diff
@@ -150,6 +150,78 @@ describe('ez-platform-ui-app', function() {
         });
     });
 
+    describe('handle contentDiscover', function () {
+        const property = 'anything';
+        let eventFired = false;
+
+        function fireContentDiscover () {
+            element.dispatchEvent(new CustomEvent('ez:contentDiscover', {
+                detail: {
+                    config: {
+                        anyProp: property,
+                    },
+                    handlers: {
+                        anyEventName: () => {
+                            eventFired = true;
+                        },
+                    },
+                },
+            }));
+        }
+
+        afterEach(function () {
+            if (document.querySelector('ez-universal-discovery')) {
+                document.querySelector('ez-universal-discovery').parentNode.removeChild(document.querySelector('ez-universal-discovery'));
+            }
+        });
+
+        beforeEach(function () {
+            fireContentDiscover();
+        });
+
+        it('should create <ez-universal-discovery>', function () {
+            const udw = document.querySelector('ez-universal-discovery');
+
+            assert.ok(
+                udw,
+                '<ez-universal-discovery> should be created'
+            );
+        });
+
+        it('should set the properties to the universal discovery widget', function () {
+            const udw = document.querySelector('ez-universal-discovery');
+
+            assert.equal(
+                udw.anyProp,
+                property,
+                'universal discovery component should get the properties given in the event'
+            );
+        });
+
+        it('should set the handlers to the universal discovery widget', function () {
```

Comment:
I'd rephrase this a bit (setting XXX to the UDW doesn't sound right). What about `should register the provided handlers as universal discovery widget listeners` ? It describes what is done better I think.

Sorry if I'm off-topic :)